tests/xfail/xfail.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package xfail
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// Enable replaces the default Fail handler with a XFail for one occurrence.
+// XFail skips the test once it fails and leaves a XFAIL label on the message.
+// It is useful to mark tests as XFail when one wants them to run even though they fail,
+// monitoring and collecting information.
+func Enable(reason string) {
+	RegisterFailHandler(func(m string, offset ...int) {
```

Comment:
https://github.com/onsi/gomega/blob/master/gomega_dsl.go#L104
in case you want to keep the suggested var names